src/app/event/[eventId]/(eventId)/rundown/RundownTable.tsx

```diff
@@ -24,12 +33,46 @@ export const RundownTable = ({ eventId }: RundownTableProps) => {
     setOpenPrompt(true)
   }
 
+  const handleDelete = async (rundownId: string) => {
+    try {
+      await deleteRundown({ id: rundownId }).unwrap()
+    } catch (error) {
+      console.error('Failed to delete the rundown', error)
```

Comment:
I think it is better to use toast to make sure user get the error message. also if there is an error message from the backend you should show that instead to give better understanding (if any). example:
```
toast.error("message here")
```

Reply:
I agree with catherine. Use toast.error() for displaying any message of potential error.